### ISSUE TITLE
pilot without skip_waiting:true appears to be broken(?)

### ISSUE THREAD

**Issue by intari** [fastlane/fastlane#8867]
### New Issue Checklist

- [X] Updated fastlane to the latest version
- [X] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description
pilot (more specifically - fastlane beta --verbose) fails to... upload with message 
(app details replaced with xxx)

`
DEBUG [2017-04-12 18:25:52.59]: [Transporter]: INFO: Done performing uploadDone notification to Apple.
DEBUG [2017-04-12 18:25:52.60]: [Transporter]: INFO: The package: /var/folders/w8/3bmht_s925n87_vwfw1t04k00000gn/T/d20170412-17779-swtinv/1225797063.itmsp has been successfully uploaded.
INFO [2017-04-12 18:25:52.77]: iTunes Transporter successfully finished its job
DEBUG [2017-04-12 18:25:52.77]: [Transporter]: DBG-X: Returning 0
INFO [2017-04-12 18:25:53.25]: -----------------------------------------------------
INFO [2017-04-12 18:25:53.25]: Successfully uploaded package to iTunes Connect. It might take a few minutes until it's visible online.
INFO [2017-04-12 18:25:53.25]: -----------------------------------------------------
INFO [2017-04-12 18:25:53.26]: Successfully uploaded the new binary to iTunes Connect
INFO [2017-04-12 18:25:53.26]: If you want to skip waiting for the processing to be finished, use the `skip_waiting_for_build_processing` option
INFO [2017-04-12 18:25:53.26]: Waiting for iTunes Connect to process the new build
INFO [2017-04-12 18:26:29.01]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:27:05.20]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:27:38.08]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:28:10.09]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:28:44.06]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:29:16.37]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:29:48.48]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:30:20.33]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:30:54.92]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:31:26.97]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:31:58.73]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:32:30.80]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:33:03.15]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:33:35.30]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:34:07.62]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:34:40.00]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:35:13.31]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:35:45.17]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:36:19.97]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:36:52.03]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:37:23.86]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:38:01.78]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:38:35.73]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:39:07.74]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:39:40.88]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:40:12.96]: Waiting for iTunes Connect to finish processing the new build (1.0 - 264)
INFO [2017-04-12 18:40:42.99]: Successfully finished processing the build
INFO [2017-04-12 18:40:42.99]: You can now tweet: 
WARN [2017-04-12 18:40:42.99]: iTunes Connect #iosprocessingtime 15 minutes
INFO [2017-04-12 18:40:44.16]: -------------------
INFO [2017-04-12 18:40:44.16]: --- Step: slack ---
INFO [2017-04-12 18:40:44.16]: -------------------
DEBUG [2017-04-12 18:40:44.18]: Taking value for 'slack_url' from environment variable 'SLACK_URL'
INFO [2017-04-12 18:40:45.35]: Successfully sent Slack notification
WARN [2017-04-12 18:40:45.35]: Lane Context:
INFO [2017-04-12 18:40:45.35]: {:DEFAULT_PLATFORM=>:ios, :PLATFORM_NAME=>:ios, :LANE_NAME=>"ios beta", :BUILD_NUMBER=>"111", :IPA_OUTPUT_PATH=>"/Users/dkzm/.jenkins/workspace/xxx - залить обновления в TestFlight/xxx.ipa", :DSYM_OUTPUT_PATH=>"/Users/dkzm/.jenkins/workspace/xxx - залить обновления в TestFlight/xxx.app.dSYM.zip", :XCODEBUILD_ARCHIVE=>"/Users/dkzm/Library/Developer/Xcode/Archives/2017-04-12/xxx 2017-04-12 18.13.03.xcarchive", :GET_INFO_PLIST_VALUE_CUSTOM_VALUE=>nil, :READ_CHANGELOG_CHANGELOG_PATH=>"./CHANGELOG.md", :READ_CHANGELOG_SECTION_CONTENT=>"\n"}
ERROR [2017-04-12 18:40:45.35]: 
INFO [2017-04-12 18:40:45.36]: Successfully generated documentation at path '/Users/dkzm/.jenkins/workspace/xxx - залить обновления в TestFlight/fastlane/README.md'

+------+-------------------------------------+-------------+
|                     fastlane summary                     |
+------+-------------------------------------+-------------+
| Step | Action                              | Time (in s) |
+------+-------------------------------------+-------------+
| 1    | Verifying required fastlane version | 0           |
| 2    | default_platform                    | 0           |
| 3    | git_pull                            | 2           |
| 4    | clear_derived_data                  | 1           |
| 5    | number_of_commits                   | 0           |
| 6    | increment_build_number              | 1           |
| 7    | commit_version_bump                 | 0           |
| 8    | gym                                 | 413         |
| 9    | upload_symbols_to_crashlytics       | 209         |
| 10   | read_changelog                      | 0           |
| 💥   | pilot                               | 1042        |
| 12   | slack                               | 1           |
+------+-------------------------------------+-------------+

DEBUG [2017-04-12 18:40:45.45]: All plugins are up to date
ERROR [2017-04-12 18:40:45.45]: fastlane finished with errors

Looking for related GitHub issues on fastlane/fastlane...
Search query: The request could not be completed because:
	

URL: https://api.github.com/search/issues?q=The%20request%20could%20not%20be%20completed%20because:%0A%09+repo:fastlane/fastlane
➡️  Deliver upload broken
   https://github.com/fastlane/fastlane/issues/8839 [open] 3 💬
   a day ago

➡️  Unexpected Error when submitting the app for review
   https://github.com/fastlane/fastlane/issues/8819 [open] 1 💬
   4 days ago

➡️  Distributing new build: The request could not be completed because:Forbidden
   https://github.com/fastlane/fastlane/issues/6023 [closed] 2 💬
   07 Dec 2016

and 119 more at: https://github.com/fastlane/fastlane/search?q=The%20request%20could%20not%20be%20completed%20because:%0A%09&type=Issues&utf8=✓

🔗  You can ⌘ + double-click on links to open them directly in your browser.

DEBUG [2017-04-12 18:40:46.82]: All plugins are up to date
/Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/spaceship/lib/spaceship/tunes/tunes_client.rb:222:in `handle_itc_response': [!] The request could not be completed because: (Spaceship::TunesClient::ITunesConnectError)
	
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/spaceship/lib/spaceship/tunes/tunes_client.rb:840:in `update_build_information!'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/spaceship/lib/spaceship/tunes/build.rb:149:in `update_build_information!'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/pilot/lib/pilot/build_manager.rb:74:in `distribute'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/pilot/lib/pilot/build_manager.rb:40:in `upload'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/actions/pilot.rb:15:in `run'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/runner.rb:252:in `block (2 levels) in execute_action'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/runner.rb:230:in `block in execute_action'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/runner.rb:226:in `chdir'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/runner.rb:226:in `execute_action'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/runner.rb:148:in `trigger_action_by_name'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/fast_file.rb:146:in `method_missing'
	from Fastfile:128:in `block (2 levels) in parsing_binding'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/lane.rb:33:in `call'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/lane.rb:33:in `call'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/runner.rb:45:in `chdir'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/runner.rb:45:in `execute'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/lane_manager.rb:52:in `cruise_lane'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/command_line_handler.rb:30:in `handle'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/commands_generator.rb:104:in `block (2 levels) in run'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.4/lib/commander/command.rb:178:in `call'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.4/lib/commander/command.rb:178:in `call'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.4/lib/commander/command.rb:153:in `run'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.4/lib/commander/runner.rb:476:in `run_active_command'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:38:in `run!'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.4/lib/commander/delegates.rb:15:in `run!'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/commands_generator.rb:303:in `run'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/fastlane/lib/fastlane/cli_tools_distributor.rb:65:in `take_off'
	from /Users/dkzm/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.26.1/bin/fastlane:20:in `<top (required)>'
	from /Users/dkzm/.fastlane/bin/bundle/bin/fastlane:22:in `load'
	from /Users/dkzm/.fastlane/bin/bundle/bin/fastlane:22:in `<main>'

`

### Environment
<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.12.4                                     |
| Ruby                        | 2.2.4                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.11.0 (Apple Git-81)           |
| Installation Source         | ~/.fastlane/bin/bundle/bin/fastlane         |
| Host                        | Mac OS X 10.12.4 (16E195)                   |
| Ruby Lib Dir                | ~/.fastlane/bin/bundle/lib                  |
| OpenSSL Version             | OpenSSL 1.0.2g  1 Mar 2016                  |
| Is contained                | false                                       |
| Is homebrew                 | true                                        |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 8.3.1                                       |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE | en_US.UTF-8 | ✅ |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# Customise this file, documentation can be found here:
# https://github.com/fastlane/fastlane/tree/master/fastlane/docs
# All available actions: https://docs.fastlane.tools/actions
# can also be listed using the `fastlane actions` command

# Change the syntax highlighting to Ruby
# All lines starting with a # are ignored when running `fastlane`

# If you want to automatically update fastlane if a new version is available:
# update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
fastlane_version "2.26.1"

default_platform :ios

platform :ios do

  before_all do
	#Configure our Slack channel. Could be used as secure var? or not? is it post-only
    ENV["SLACK_URL"] = "https://hooks.slack.com/services/xxxx"
	#get git tags
	git_pull(only_tags: true) # do we need it ?
    #cocoapods

  end




  desc "Bun tests"
  desc "This will also make sure the profile is up to date"
  lane :test do
 	 clear_derived_data

	 #TODO:increase number of devices we test?
  	 scan(scheme: "xxx",
       output_types: "",
       #devices: ['iPhone 6', 'iPhone 6 Plus','iPhone 5', 'iPhone 4s'],
       devices: ['iPhone 5'],
       code_coverage: true,
       fail_build: false
     )

  	 trainer(output_directory: "./fastlane")

     xcov(workspace: "xxx.xcworkspace",
  		scheme: "xxx",
  		output_directory: "xcov_output",
  		only_project_targets: true,
  		json_report: true,
  		html_report: true,
  		markdown_report: true,
  		slack_url:ENV["SLACK_URL"]
	 )

     clean_build_artifacts

	 #scan and xcov will send updates arleady
	 slack(
     	 message: "Tests pass for xxx",
      	 channel: "#general"
     )

	  # sh "your_script.sh"
      # You can also use other beta testing services here (run `fastlane actions`)

  end


  desc "Build and Submit a new Beta Build to Apple TestFlight using Jenkins"
  desc "This will also make sure the profile is up to date"
  lane :beta do


	  clear_derived_data


	  #use number of commits as build number
	  build_number = number_of_commits(all: true)
	  increment_build_number(build_number: build_number)

	  commit_version_bump(
 	  	message: "Version Bump",# create a commit with a custom message
  	  	xcodeproj: "xxxx.xcodeproj", # optional, if you have multiple Xcode project files, you must specify your main project here
	  )


      gym(scheme: "xxxx", export_method: "app-store") # Build your app - more options available
      upload_symbols_to_crashlytics # upload symobls to Crashlytics

	  changelog = read_changelog # Read changelog
      pilot(changelog: changelog) # Send binary and changelog to TestFlight
	  #pilot(skip_waiting_for_build_processing: true) # send build to TestFlight and don't wait until iTunes Connect finishes

      tag_for_changelog="fastlane-beta/iosbeta/b#{build_number}"

      stamp_changelog(section_identifier: "Build #{build_number}",
      	git_tag: tag_for_changelog
      ) # Stamp Unreleased section with newly released build number

	  git_commit(path: ["./CHANGELOG.md"], message: "Updated chanelog for build #{build_number}")

	  # add new tag for this build
 	  add_git_tag(
  		grouping: "fastlane-beta",
  		prefix: "b"
	  )


      rocket
	  clean_build_artifacts

	  emojified_changelog = emojify_changelog # Emojify the output of `read_changelog` action
      slack(message: "New beta release, which includes the following: #{emojified_changelog}",
      	channel: "#general") # share on Slack

	  # stamp_changelog changed local git. we also add tag (but 'all done' part take car of tag)
	  push_to_git_remote

	  # sh "your_script.sh"
      # You can also use other beta testing services here (run `fastlane actions`)

  end


  #Just do screenshots and upload them to iTunes Connect
  lane :screenshots do
    snapshot # Generate screenshots (using special Simulator script)
    deliver(skip_binary_upload:true) # Deliver meta and screenshots (but not binary) to iTunes Connect
    # Send message to Slack
	slack(
    	 message: "Sent new screenshots and metadata of xxx to AppStore",
      	 channel: "#general"
    )

  end


  after_all do |lane|
    # This block is called, only if the executed lane was successful

    # update tags if we did any changes
    push_git_tags
    #slack(
    #   message: "Successfully deployed new App Update."
    #)
  end

  error do |lane, exception|
    slack(
       message: exception.message,
       success: false,
       channel: "#general"
    )
  end
end



# More information about multiple platforms in fastlane: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Platforms.md
# All available actions: https://docs.fastlane.tools/actions

# fastlane reports which actions are used
# No personal data is recorded. Learn more at https://github.com/fastlane/enhancer
```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier "xxxxx.prod" # The bundle identifier of your app
apple_id "xxx@yyy.com" # Your Apple email address

team_id "xxx" # Developer Portal Team ID

# you can even provide different app identifiers, Apple IDs and team names per lane:
# More information: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Appfile.md

itc_team_name "xxxx"
```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.26.1  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                    | Version | Update-Status |
| ------------------------- | ------- | ------------- |
| fastlane-plugin-trainer   | 0.4.0   | ✅ Up-To-Date  |
| fastlane-plugin-changelog | 0.7.0   | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| slack-notifier            | 1.5.1        |
| i18n                      | 0.7.0        |
| json                      | 1.8.1        |
| thread_safe               | 0.3.5        |
| tzinfo                    | 1.2.2        |
| minitest                  | 5.4.3        |
| activesupport             | 4.2.7.1      |
| CFPropertyList            | 2.3.5        |
| claide                    | 1.0.1        |
| colored                   | 1.2          |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.4.2        |
| rouge                     | 1.11.1       |
| xcpretty                  | 0.2.4        |
| terminal-notifier         | 1.7.1        |
| unicode-display_width     | 1.1.3        |
| terminal-table            | 1.7.3        |
| plist                     | 3.2.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.0        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.5.0        |
| babosa                    | 1.0.2        |
| highline                  | 1.7.8        |
| commander-fastlane        | 4.4.4        |
| excon                     | 0.54.0       |
| faraday                   | 0.11.0       |
| unf_ext                   | 0.0.7.2      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20161129 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| fastimage                 | 2.0.1        |
| gh_inspector              | 1.0.3        |
| uber                      | 0.0.15       |
| representable             | 2.3.0        |
| retriable                 | 2.1.0        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| hurley                    | 0.2          |
| little-plugger            | 1.1.4        |
| multi_json                | 1.12.1       |
| logging                   | 2.1.0        |
| jwt                       | 1.5.6        |
| memoist                   | 0.15.0       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.1        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.9.26       |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.0        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 0.0.4        |
| dotenv                    | 2.2.0        |
| bundler                   | 1.14.3       |
| faraday_middleware        | 0.11.0.1     |
| trainer                   | 0.7.0        |
| fastlane-plugin-trainer   | 0.4.0        |
| fastlane-plugin-changelog | 0.7.0        |
</details>

*generated on:* **2017-04-12**
</details>

I have 'application manager' under organization's iTunes Connect account.
I clicked 'confirm' and checked 'do not show' on today's message about TestFlight in iTunes connect.
IF I comment out pilot(changelog:...) and replace it with pilot(skip_waiting:true) - everything works. 
Without waiting and without sending changelogs of course.

**Comment by mattoakesnet** [comment]
Likely related to the new Testflight features https://developer.apple.com/news/?id=04112017a

**Comment by KrauseFx** [comment]
We're working on migrating _pilot_ to use the latest API, you can follow the progress here https://github.com/fastlane/fastlane/pull/8871

Unfortunately, there are still some bugs on the back-end we have to wait for to be resolved before we can proceed.

**Comment by erikvillegas** [comment]
I think this is also new in iTunes Connect TestFlight:
<img width="549" alt="screen shot 2017-04-13 at 11 55 39 am" src="https://cloud.githubusercontent.com/assets/1296942/25019728/7d9699fe-2040-11e7-9171-65d6d60900a9.png">

<img width="607" alt="screen shot 2017-04-13 at 11 59 02 am" src="https://cloud.githubusercontent.com/assets/1296942/25019755/9bfc45ba-2040-11e7-9708-ddfc0974ec69.png">

Perhaps pilot needs some export compliance options like delivery does.

**Comment by mpirri** [comment]
Another update: we have found a new method of submitting app builds to TestFlight and are adding this to `pilot`. Hope to have another update on this shortly.

Thanks for your patience!

**Comment by KrauseFx** [comment]
Thanks everybody for reporting this. We're busy working on migrating _pilot_ to use the new TestFlight. Unfortunately it requires a big refactor from our side, and it will probably take another week. Please subscribe to https://github.com/fastlane/fastlane/pull/8871 to stay up to date 👍

**Comment by petersuwara** [comment]
Thanks for being on top of this guys. Apple really through us a curve ball on this with the Test Flight changes.

**Comment by mpirri** [comment]
@intari Can you confirm if this issue is still happening with the latest version of _fastlane_? I'm optimistic that the changes in `pilot` to support the new TestFlight should have fixed this.

Thanks much!

**Comment by biterchus** [comment]
Hi!, it would be nice to have be able to set the export compliance checks, as I currently have CI pipeline broken, as I have to manually provide that information to TestFlight :(

Thanks.

**Comment by intari** [comment]
@mpirri I can confirm that this issue does not happen on 2.30.2

**Comment by mpirri** [comment]
@intari Great news - I'll close this ticket then.  Thanks for the update!

**Comment by nelsballs** [comment]
@intari Have you just added `<key>ITSAppUsesNonExemptEncryption</key><false/>` to your Info.plist before running fastlane ? I'm using 2.36 and I still have to manually check false on ITC to start it on TestFlight : /

**Comment by intari** [comment]
@nelsballs I added this key long ago. I don't saw any issues with it but right now I only use Fastlane with Internal Testflight

**Comment by nelsballs** [comment]
@intari I'm also running it with internal testing, are you also on 2.36 ?
